lint.ignore

```diff
@@ -49,6 +49,27 @@ TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.sxg
 ## .gitignore
 W3C-TEST.ORG: .gitignore
 
+## Licenses ##
+
+# The 3-Clause BSD License
+LICENSE: LICENSE.md
+
+# Various license files. Do not add to this list without going through the RFC
+# process: https://github.com/web-platform-tests/rfcs
+LICENSE: compression/pako/LICENSE
+LICENSE: css/css-color/LICENSE
+LICENSE: css/css-ui/support/PTS/PngSuite.LICENSE
+LICENSE: css/CSS2/LICENSE-BSD
```

Comment:
I've sent https://github.com/web-platform-tests/wpt/pull/23593 to remove these 3 files.